pkg/registry/constants.go

```diff
@@ -42,7 +45,7 @@ const (
 // KnownMediaTypes returns a list of layer mediaTypes that the Helm client knows about
 func KnownMediaTypes() []string {
```

Comment:
Should `HelmChartConfigMediaType` be a known media type?

Reply:
Just for the layer types. `HelmChartMetaMediaType` not removed, the var name was just changed